### PR TITLE
Add Romanian language

### PR DESCRIPTION
**Pull request by liviuroman**

### CODE REVIEW

**Review by likerRr — CHANGES_REQUESTED** [review]
Thank you! Please, add tests like it's done for [other languages](https://github.com/hustcc/timeago.js/tree/master/tests/locales)

**Comment by hustcc** [comment]
``` js
var timeType = parseInt((index + 1) / 2);
```

This can do, no need to switch ... case.

**Comment by likerRr** [comment]
@hustcc not sure that we can replace switch/case with such code. Compare results for case 2 and 3:

``` javascript
parseInt((2 + 1) / 2); // 1
parseInt((3 + 1) / 2); // 2
// but result should be 2 in both cases
```

@liviuroman Hope, that `Math.ceil((index + 1) / 2)` can help:

``` javascript
Math.ceil((1 + 1) / 2); // 1

Math.ceil((2 + 1) / 2); // 2
Math.ceil((3 + 1) / 2); // 2

Math.ceil((4 + 1) / 2); // 3
Math.ceil((5 + 1) / 2); // 3

Math.ceil((6 + 1) / 2); // 4
Math.ceil((7 + 1) / 2); // 4

Math.ceil((8 + 1) / 2); // 5
Math.ceil((9 + 1) / 2); // 5

Math.ceil((10 + 1) / 2); // 6
Math.ceil((11 + 1) / 2); // 6

Math.ceil((12 + 1) / 2); // 7
Math.ceil((13 + 1) / 2); // 7
```

File: locales/ro.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        case 4:
+        case 5: timeType = 3; break;
+        case 6:
+        case 7: timeType = 4; break;
+        case 8:
+        case 9: timeType = 5; break;
+        case 10:
+        case 11: timeType = 6; break;
+        case 12:
+        case 13: timeType = 7; break;
```
Comment (likerRr) [inline locales/ro.js]: [Simplify switch/case](https://github.com/hustcc/timeago.js/pull/75#issuecomment-249110364)

File: locales/ro.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    ['un minut',    '%s minute',    '%s de minute' ],   // Minutes
+    ['o oră',       '%s ore',       '%s de ore'],       // Hours
+    ['o zi',        '%s zile',      '%s de zile'],      // Days
+    ['o săptămână', '%s săptămâni', '%s de săptămâni'], // Weeks
+    ['o lună',      '%s luni',      '%s de luni'],      // Months
+    ['un an',       '%s ani',       '%s de ani']        // Years
+];
+
+function formatTime(type, number) {
+    if (number == 1)
```
Comment (likerRr) [inline locales/ro.js]: Wrap `if`s in curly braces until it's one-line `if`

**Review by likerRr — CHANGES_REQUESTED** [review]
Please, fix remaining parts

File: locales/ro.js (hunk 1 above)
Comment (hustcc) [reply locales/ro.js]: :+1:  How to use this Github feature ?

Comment (likerRr) [reply locales/ro.js]: @hustcc open tab "Files changed" and find green button "Review Changes". Read more [here](https://github.com/blog/2256-a-whole-new-github-universe-announcing-new-tools-forums-and-features) or view a [video](https://www.youtube.com/watch?v=HW0RPaJqm4g).

**Comment by likerRr** [comment]
@liviuroman thank you for updates. Resolve conflicts and we will approve the PR.

**Comment by likerRr** [comment]
@liviuroman sorry, but conflicts still haven't resolved. I think it would be better to close this PR and open a new one, because there are already 9 commits for just adding new locale. It's too much. 

Please, get latest changes from timeago's `master`, apply [patch](https://patch-diff.githubusercontent.com/raw/hustcc/timeago.js/pull/75.patch) from this PR, resolve conflicts (please, done this all locally) and then open a new PR with single commit. Thank you!

**Pull request closed without merging**